source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,190 @@
+
+.. code-block:: bash
+
+	dig example.com
+
+While the response should be the same, the ``SERVER`` in the response should look like ``;; SERVER: 127.0.0.53#53(127.0.0.53)``. Note that the final IPv4 digit is 53 and not 1, as with our Unbound instance.
+
+Setting up for a single machine
+-------------------------------
+
+Now that we have configured and tested our Unbound server, we can tell our machine to use it by default. The nameserver (i.e. resolver) your machine uses by default is defined in :file:`/etc/resolv.conf`.
+While just changing this file will work as long as the machine doesn't reboot, the more permanent and better solution is to replace the file with our own. The reason for this is that the :file:`resolv.conf` file is a `symbolic link`. We will remove the link and create a new file ourselves.
+
+.. code-block:: bash
+
+	rm /etc/resolv.conf
+
+With your favourite text editor (if you don't have a favourite you could use :command:`nano`), we can then create a new file with the same name and fill it with the IP address that our Unbound instance is running at, and we include the :option:`edns0` option as this enables header extensions used in DNSSEC and is an overall standard used in DNS nowadays. |br|
+So with :file:`nano /etc/resolv.conf` we create the new file and enter:
+
+.. code-block:: bash
+
+	nameserver 127.0.0.1
+	options edns0
+
+We then need to stop and disable the currently running pre-installed resolver. Note that you lose connectivity to the web until the next step.
+
+.. code-block:: bash
+
+	sudo systemctl disable systemd-resolved.service
+	sudo systemctl stop systemd-resolved
+
+Now the operating system should use our Unbound instance as default. A quick test a :command:`dig` without specifying the address of the Unbound server should give the same result as specifying it did above (with ``@127.0.0.1``).
+
+.. code-block:: bash
+
+	dig example.com
+
+Note that the "SERVER" section in the output from :command:`dig` should also contain the local IP address of our server.
+
+.. code-block:: bash
+
+	;; SERVER: 127.0.0.1#53(127.0.0.1)
+
+Setting up for the rest of the network
+--------------------------------------
+
+While we currently have a working instance of Unbound, we need it to be reachable from within our entire network. With that comes the headache of dealing with (local) IP addresses. It’s likely that your home router distributed local IP addresses to your devices. If this is the case (i.e. you didn’t change it by hand), they should be :rfc:`1918` ranges:
+
+.. code-block:: bash
+
+	10.0.0.0 - 10.255.255.255 (10/8)
+	172.16.0.0 - 172.31.255.255 (172.16/12)
+	192.168.0.0 - 192.168.255.255 (192.168/16)
+
+To find the IP address of the machine that is running Unbound, we use:
+
+.. code-block:: bash
+
+	hostname -I
```

Comment:
While `-I` is perfectly fine, I like using the long option so it's clearer what it does; in this case `--all-ip-addresses`